viewmaster/viewmaster.py

```
@@ -152,52 +228,39 @@
-        abspaths = [p.strip() for p in abspaths]
-        return abspaths
+    Raises:
+        IOError: If the PDS3_HOLDINGS_DIR environment variable is not set.
+    """
 
-    raise IOError('HOLDINGS_PATHS not found in httpd_customization.conf')
+    pds3_holdings_dir = os.getenv('PDS3_HOLDINGS_DIR')  # XXX PDS4
+    if pds3_holdings_dir is not None:
+        return [pds3_holdings_dir]
+    else:
+        raise IOError("'PDS3_HOLDINGS_DIR' environment variable not set")
```

Ruff (0.14.11)

Replace aliased errors with `OSError`

Replace `IOError` with builtin `OSError`

(UP024)